lib/matplotlib/_docstring.py

```diff
@@ -3,6 +3,28 @@
 from . import _api
 
 
+def kwdoc(text):
```

Comment:
This name and description is still unclear to me, even after you explained it to me.  I think the name should be more descriptive, and the docstring here more explicit about what problem this solves.

Reply:
The name was taken over from the function that generates uses this information to generate the docstring we insert for the kwargs:

https://github.com/matplotlib/matplotlib/blob/2f8fc76e45d6ca8359bb88cda4b39fcaeaabd3e3/lib/matplotlib/artist.py#L1779

But I agree, it's a bit mysterious. Would this be better?

```
def kwarg_doc(text):
    Decorator for defining the documentation used when artist properties are
    passed through as keyword arguments.

    See e.g. the `**kwargs` section in `.Axes.text`.

```

Reply:
I still think the above would be more clear

Reply:
Chaned to `kwarg_doc`.